src/v2/Apps/Consign/Components/InDemandNow/ConsignInDemandNow.tsx

```diff
@@ -1,81 +1,385 @@
-              medium
-              mediumHighRangeCents
-              mediumLowRangeCents
-              mediumMidRangeCents
-              midRangeCents
-              sellThroughRate
-              smallHighRangeCents
-              smallLowRangeCents
-              smallMidRangeCents
-              updatedAt
+    <QueryRenderer<ConsignInDemandNowQuery>
+      environment={relayEnvironment}
+      variables={{
+        artistInternalId: "4ed901b755a41e0001000a9f", // Kehinde Wiley
+        artistSlug: "kehinde-wiley",
+        medium: "PAINTING",
+      }}
+      // FIXME: Must be logged in for diffusion data to work
+      query={graphql`
+        query ConsignInDemandNowQuery(
+          $artistInternalId: ID!
+          $artistSlug: String!
+          $medium: String!
+        ) {
+          artist(id: $artistSlug) {
+            birthday
+            nationality
+            auctionResultsConnection(first: 1, sort: DATE_DESC) {
+              edges {
+                node {
+                  internalID
+                  title
+                  dimensionText
+                  images {
+                    thumbnail {
+                      url
+                    }
+                  }
+                  description
+                  dateText
+                  organization
+                  saleDate
+                  priceRealized {
+                    display
+                    centsUSD
+                  }
+                }
+              }
             }
           }
-        `}
-        render={({ props, error }) => {
-          // FIXME: Error handling
-          if (error) {
-            return null
-          }
-          // FIXME: Add skeleton loading state
-          if (!props) {
-            return null
+
+          marketPriceInsights(artistId: $artistInternalId, medium: $medium) {
+            annualLotsSold
+            annualValueSoldCents
+            artistId
+            artistName
+            artsyQInventory
+            createdAt
+            demandRank
+            demandTrend
+            highRangeCents
+            largeHighRangeCents
+            largeLowRangeCents
+            largeMidRangeCents
+            liquidityRank
+            lowRangeCents
+            medianSaleToEstimateRatio
+            medium
+            mediumHighRangeCents
+            mediumLowRangeCents
+            mediumMidRangeCents
+            midRangeCents
+            sellThroughRate
+            smallHighRangeCents
+            smallLowRangeCents
+            smallMidRangeCents
+            updatedAt
           }
+        }
+      `}
+      render={({ props, error }) => {
+        // FIXME: Error handling
+        if (error) {
+          return null
+        }
+
+        return (
+          <>
+            <Media greaterThanOrEqual="md">
+              <InDemandNowLarge {...props} />
+            </Media>
+            <Media lessThan="md">
+              <InDemandNowSmall {...props} />
+            </Media>
```

Comment:
The duplication below is necessary because the layouts are significantly different between desktop / mobilew